examples/sui-move/simple_warrior/sources/example.move

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Demonstrates wrapping objects using the `Option` type.
+module simple_warrior::example {
+    use std::option::{Self, Option};
+    use sui::object::{Self, UID};
+    use sui::tx_context::TxContext;
+
+    struct Sword has key, store {
+        id: UID,
+        strength: u8,
+    }
+
+    struct Warrior has key {
```

Comment:
I think we need `store` here, to transfer it to the user (in a tx on the PTB), since there's no module-specific transfer function!

Reply:
Ah yes, will add, thanks!